app/models/floating_ip.rb

```diff
@@ -7,18 +7,14 @@ class FloatingIp < ApplicationRecord
   # TODO(lsmola) NetworkManager, once all providers use network manager rename this to
   # "ManageIQ::Providers::NetworkManager"
   belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::BaseManager"
-  # TODO(lsmola) NetworkManager, remove when all providers share the new network architecture
+  # Not going through network_port because of Amazon, old EC2 way allows to associate public Ip to instance, without
+  # any network_port used
```

Comment:
I think it might be in Azure too, maybe in GCE also

The UI table floating_ips shows vm name, so it does .joins(:vm), that means there has to be working relationship in base model, so floating_ips of all providers show up correctly.